site/docs/contribute/modules.md

```diff
@@ -92,6 +134,9 @@ If the module workload needs to return information to the user, that information
 
 For a full example see the [Arrow Flight Module chart](https://github.com/fybrik/arrow-flight-module/tree/master/helm/afm).
 
+> **NOTE**: Helm values that are passed from Fybrik to the modules, override the default values defined in the values.yaml file.  
+To add additional parameters to be passed to the module, it is recommended to use and configure a conf.yaml file.  
```

Comment:
To add additional parameters to be passed to the module, it is recommended to use the conf.yaml file.